internal/obsreportconfig/obsreportconfig.go

```diff
@@ -31,6 +31,14 @@ var UseOtelForInternalMetricsfeatureGate = featuregate.GlobalRegistry().MustRegi
 	featuregate.StageAlpha,
 	featuregate.WithRegisterDescription("controls whether the collector uses OpenTelemetry for internal metrics"))
 
+// DisableHighCardinalityMetricsfeatureGate is the feature gate that controls whether the collector should enable
+// potentially high cardinality metrics.
+var DisableHighCardinalityMetricsfeatureGate = featuregate.GlobalRegistry().MustRegister(
```

Comment:
Feature flags are not for configuration. Please describe how you intend to remove this?

Reply:
This was discussed here https://github.com/open-telemetry/opentelemetry-collector/issues/7517#issuecomment-1511168350 it is meant as an interim solution until we can configure views on the OTel configuration, at which point we can remove this.

Reply:
Please document that in the gate description to set the expectations.